test/functional/apps/discover/_data_grid_doc_table.ts

```diff
@@ -101,6 +105,55 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       );
     });
 
+    it('should show popover with expanded cell content by click on expand button on embeddable', async () => {
+      log.debug('open popover with expanded cell content to get json from the editor');
+      await PageObjects.common.navigateToApp('discover');
```

Comment:
since you navigate to Discover via `beforeEach` this should be redundant, right?